gradle/libs.versions.toml

```diff
@@ -25,8 +25,8 @@ auto-value-annotations = "com.google.auto.value:auto-value-annotations:1.10.4"
 checkstyle = "com.puppycrawl.tools:checkstyle:10.12.5"
 commons-math3 = "org.apache.commons:commons-math3:3.6.1"
 conscrypt = "org.conscrypt:conscrypt-openjdk-uber:2.5.2"
-cronet-api = "org.chromium.net:cronet-api:113.5672.61"
-cronet-embedded = "org.chromium.net:cronet-embedded:113.5672.61"
+cronet-api = "org.chromium.net:cronet-api:108.5359.79"
+cronet-embedded = "org.chromium.net:cronet-embedded:108.5359.79"
```

Comment:
I just realized I didn't add the notes here from https://github.com/grpc/grpc-java/pull/10359 here. I've even created a ticket for it https://github.com/grpc/grpc-java/issues/10396. Will Add the comment once we merge your PR.